data/json/items/gunmod/sights.json

```diff
@@ -37,14 +37,14 @@
   {
     "id": "improve_sights",
     "type": "GUNMOD",
-    "name": "improved iron sights",
+    "name": "iron sights",
```

Comment:
Change here is for consistency with `name_plural`. At present this item offers no improvement over standard sights but they are currently useful when a less conventional (especially home made) has poor sights. 

Whatever we go with `name` and `name_plural` should be equivalent. If we want them to be an improvement then we can look at that in a separate PR.